src/components/Card.js

```diff
@@ -72,6 +101,16 @@ export default class Card {
     //set description
     this._cardElement.querySelector(".card__title").textContent = this._name;
 
+    if (this._isLiked === true) {
```

Comment:
Needs correction: there should not be code duplication, use `_renderLikes` method